Examples/php/enum/runme.php

```diff
@@ -23,10 +21,10 @@
 enum_test(1234,5678);
 
 print "\nTesting use of enum with class method\n";
-$f = new_Foo();
+$f = new Foo();
 
-Foo_enum_test($f,Foo_IMPULSE);
-Foo_enum_test($f,Foo_WARP);
-Foo_enum_test($f,Foo_LUDICROUS);
+$f->enum_test(Foo_IMPULSE);
+$f->enum_test(Foo_WARP);
+$f->enum_test(Foo_LUDICROUS);
```

Comment:
So currently this example explicitly builds with `-noproxy`, so this change will stop it working.

I think we probably should use the class wrappers instead for all the examples, so this change seems OK.  But it ought to also change to use `Foo::IMPULSE`, etc as well since those are the class wrapped constants.